Modix/Modules/ModerationModule.cs

```diff
@@ -91,17 +92,21 @@ await ReplyAsync(logChannels.Any()
                 string reason)
             => ModerationService.CreateInfractionAsync(InfractionType.Mute, subject.Id, reason, null);
 
-        [Command("mute")]
+        [Command("tempmute")]
         [Summary("Mute a user, for a temporary amount of time.")]
         public Task TempMute(
             [Summary("The user to be muted.")]
                 IGuildUser subject,
             [Summary("The duration of the mute.")]
-                TimeSpan duration,
+                string durationString,
             [Summary("The reason for the mute.")]
             [Remainder]
                 string reason)
-            => ModerationService.CreateInfractionAsync(InfractionType.Mute, subject.Id, reason, duration);
+        {
+            var duration = XmlConvert.ToTimeSpan(durationString);
```

Comment:
`XmlConvert`?

Reply:
The only way to parse an ISO8601 duration string. I'd love to replace with a parser that handles something more like Humanizer, but unfortunately nothing like that seems to exist. Plenty of solutions for serializing in such a fashion, but not parsing.

Reply:
I had no idea 8601 had a specification for timespans -- and that I was using them at work and wrote my own parser...